### DISCUSSION TITLE
type of base image

### DISCUSSION THREAD

**Discussion by developer992** [minio/minio#20059] [Q&A] [answered]
Hello,

i am having some problems with networking and availability of containers on my setup 

i am trying to install ping and ifconfig

but there is no apt nor yum .... which image base does minio use?

Thanks

**Comment by developer992** [comment]
ok i managed to fix the networking via docker-compose but i would still like to know ...

many thanks

**Comment by marktheunissen** [comment, marked as answer]
Hi @developer992, please see: https://github.com/minio/minio/blob/master/Dockerfile.release#L33

`FROM registry.access.redhat.com/ubi9/ubi-micro:latest`

**Reply by developer992** [reply]
Ah ... thank you.
